runtime/moonbeam/src/lib.rs

```diff
@@ -752,6 +752,13 @@ impl pallet_parachain_staking::OnInactiveCollator<Runtime> for OnInactiveCollato
 type MonetaryGovernanceOrigin =
 	EitherOfDiverse<EnsureRoot<AccountId>, governance::custom_origins::GeneralAdmin>;
 
+pub struct ParaBlockNumberProvider;
```

Comment:
Add a comment here to explain that it's a temporary type that we should remove when we will enable async backing.
Also choosse a more explicit name, like `StakingRoundSlotProvider`